drivers/espi/espi_mchp_xec.c

```diff
@@ -0,0 +1,822 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <soc.h>
+#include <errno.h>
+#include <espi.h>
+#include "espi_utils.h"
+
+#define M2S             1ul
+#define S2M             0ul
+#define SMVW_ID_MAX     10ul
+
+#define SRC_ID0         0
+#define SRC_ID1         1
+#define SRC_ID2         2
+#define SRC_ID3         3
+#define SRC_ID_MAX      4
+
+struct espi_isr {
+	u32_t girq_bit;
+	void (*the_isr)(struct device *dev);
+};
+
+struct espi_xec_config {
+	u32_t base_addr;
+	u8_t bus_girq_id;
+	u8_t vw_girq_id;
+	u8_t pc_girq_id;
+};
+
+struct espi_xec_data {
+	sys_slist_t callbacks;
+	u8_t plt_rst_asserted;
+	u8_t espi_rst_asserted;
+	u8_t sx_state;
+};
+
+struct xec_signal {
+	u8_t xec_reg_idx;
+	u8_t bit;
+	u8_t dir;
+};
+
+enum MCHP_MSVW_REGS {
```

Comment:
enum main name are in small letters (and same as macros: prefix them)

Reply:
this comment has not be applied.

Reply:
@tbursztyka  addressed in most recent submission